src/pages/ValidateConfigFiles/ServiceTemplatesTab.js

```diff
@@ -307,6 +307,31 @@ class ServiceTemplatesTab extends Component {
     );
   }
 
+  async handleSesUpload(file) {
+    this.setState({ loading: true });
+    let fileContents = await readFile(file);
+    await postJson('/api/v2/service/files/ses/ses_config.yml', JSON.stringify(fileContents));
```

Comment:
I believe that `JSON.stringify` is not needed since `fileContents` is already a string.

Reply:
The way that other file content is posed in this file is the [same](https://github.com/ArdanaCLM/ardana-installer-ui/blob/23c9dc2e2e3876af914a897b36d7a06adb61f41f/src/pages/ValidateConfigFiles/ServiceTemplatesTab.js#L62). Removing the `JSON.stringify` call causes the `ardana-service` to return (400):
```
The browser (or proxy) sent a request that this server could not understand
```

Reply:
Thanks, that's good to know.